WeakAurasOptions/AuthorOptions.lua

```diff
@@ -1636,32 +1490,239 @@ local function addUserModeOption(options, config, args, data, order, i)
   if optionClass == "simple" then
     userOption = {
       type = optionType,
-      name = nameUser(data, config, option),
-      desc = descUser(data, option),
+      name = nameUser(option),
+      desc = descUser(option),
       width = (option.width or 1) * WeakAuras.normalWidth,
-      order = order,
-      get = getUser(data, config, option),
-      set = setUser(data, config, option)
+      order = order(),
+      get = getUser(option),
+      set = setUser(data, option)
     }
   elseif optionClass == "noninteractive" then
     userOption = {
       type = "description",
-      order = order,
+      order = order(),
       name = "",
-      width = (option.width or 1) * WeakAuras.normalWidth,
+      width = (option.width or 1) * WeakAuras.normalWidth
     }
+  elseif optionClass == "group" then
```

Comment:
This block (and its corresponding user mode block) are the only real “additions” in this pull request.